src/command_modules/azure-cli-eventhubs/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.2.5
++++++
+* exception handling fixed
```

Comment:
'Fix exception handling'

Reply:
Removed, has reverted the exception handling changes as core now handles bad request